src/cascadia/TerminalSettingsModel/ActionMapSerialization.cpp

```diff
@@ -105,6 +107,28 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
                 // any existing keybinding with the same keychord in this layer will get overwritten
                 _KeyMap.insert_or_assign(keys, idJson);
+
+                if (!_changeLog.contains(KeysKey.data()))

```

Comment:
Elsewhere you wrote `_changeLog.emplace(KeysKey);`. Does this not support the contains call without the `.data()` call?

Reply:
Nope :/. Even with the transparent comparators